src/main/java/calculator/domain/InputStringSeparator.java

```diff
@@ -0,0 +1,108 @@
+package calculator.domain;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public class InputStringSeparator {
+    private static final int CUSTOM_SEPARATOR_START_INDEX = 2;
```

Comment:
컨벤션에 따라 다르겠지만 대부분 static과 인스턴스 필드는 구분하기 위해 한칸 공백을 두면 좋습니다.
```suggestion
    private static final int CUSTOM_SEPARATOR_START_INDEX = 2;
    
```